_posts/en/newsletters/2023-01-04-newsletter.md

```diff
@@ -0,0 +1,200 @@
+  transactions affecting the wallet's balance.  Bitcoin Core is able to
+  determine the age of the wallet because its backup contains the date
+  the wallet was created.
+
+- [Bitcoin Core #23319][] updates the `getrawtransaction` RPC to provide
+  additional information if the `verbose` parameter is set to `2`.  The
+  additional information includes the fee the transaction paid and
+  information about each of the outputs from previous transactions
+  ("prevouts") which are spent by being used as inputs to this
+  transaction.  See [Newsletter #172][news172 prevout] for details about
+  the method used to retrieve the information.
+
+- [Bitcoin Core #26628][] begins rejecting RPC requests that include the
+  same parameter name multiple times.  Previously, the daemon treated a
+  request with repeated parameters as if it only had the last of the repeated
+  parameters, e.g. `{"foo"="bar", "foo"="baz"}` was treated as
+  `{"foo"="baz"}`.  Now the request will fail.  When using `bitcoin-cli`
+  with named parameters, the behavior is unchanged: repeated parameters
+  will not be rejected but only the last of the repeats will be sent.
+
+- [Eclair #2464][] adds the ability to trigger an event when a remote
+  peer becomes ready to process payments.  This is especially useful in
+  the context of [async payments][topic async payments] where the local
+  node temporarily holds a payment for a remote peer, waits for the peer
+  to connect (or reconnect), and delivers the payment.
+
+- [Eclair #2482][] allows sending payments using [blinded routes][topic
+  rv routing], which are paths whose last several hops are chosen
+  by the receiver.  The receiver uses onion encryption to obfuscate the
+  hop details and provides the encrypted data to the spender, along with
+  the identity of the first node along the blinded route.  The spender
+  then constructs a payment path to that first node and includes the
+  encrypted details for the operators of the last several nodes to
+  decrypt and use to forward the payment to the receiver.  This allows
+  the receiver to accept a payment without disclosing the identity of
+  their node or channels to the spender, improving privacy.
+
+- [LND #2208][] begins preferring different payment paths depending on
+  the maximum capacity of a channel relative to the amount to be spent.
+  As the amount to be sent approaches the capacity of a channel, that
+  channel becomes less likely to be selected for a path.  This is
+  broadly similar to pathfinding code already used in Core Lightning and
+  LDK.
+
+- [LDK #1738][] and [#1908][ldk #1908] provide additional features for handling
+  [offers][topic offers].
+
+- [Rust Bitcoin #1467][] adds methods for calculating the size in
+  [weight units][] of transaction inputs and outputs.
+
+- [Rust Bitcoin #1330][] removes the `PackedLockTime` type, requiring
+  code instead use the almost-identical `absolute::LockTime` type.  A
```

Comment:
I think if the code part of this sentence is dropped it reads easier

```suggestion
- [Rust Bitcoin #1330][] removes the `PackedLockTime` type, requiring
  code use of the almost-identical `absolute::LockTime` type.  A
```

Reply:
Changed to "downstream code".  I want to communicate that this isn't just an internal change, but a change that anyone depending on Rust Bitcoin for locktimes may need to account for (although hopefully it's something that will be caught by the linker when they next upgrade their dependencies, so this is just advance notice).